core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -17,7 +17,8 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
 
   public workflowExecutionsList: WorkflowExecutionsEntry[] | undefined;
 
-  public executionsTableHeaders: string[] = ["Execution#", "Starting Time", "Updated Time", "Status"];
+  public executionsTableHeaders: string[] = ["Execution#", "Starting Time", "Updated Time", "Status", "Bookmarked?"];
+  public currentlyHoveredExecution: WorkflowExecutionsEntry | null = null;
```

Comment:
Make it undefined instead of null

Reply:
I mean okay sure, but this doesn't make as much sense because the meaning of no here isn't undefined, it's there actually is nothing we are hovering over. My understanding was that undefined is used in the cases where the value is not known, but here we are either hovering over something or not.